express-server/src/users/users.js

```diff
@@ -4,8 +4,15 @@ var express = require('express');
 var router = express.Router();
 router.get('/token', async (req, res, next) => {
     const token = req.headers.authorization.replace('Bearer ', '');
-    const response = await FirebaseOperations.login(token);
-    res.send(response);
+    try {
```

Comment:
What if I call another route from the backend? Is it going to be validated for token?

Reply:
Resolved in a new branch